<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="DirectoryLabel">
            <property name="text">
             <string>Каталог</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="FileMaskLabel">
            <property name="text">
             <string>Маска файлов</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="OutputDirectoryLabel">
            <property name="text">
             <string>Выходной каталог</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="DirectoryInput"/>
          </item>
          <item>
           <widget class="QLineEdit" name="FileMaskInput"/>
          </item>
          <item>
           <widget class="QLineEdit" name="OutputDirectoryInput"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="DeleteCheckBox">
        <property name="styleSheet">
         <string notr="true">QCheckBox:unchecked{ color: black; }
QCheckBox:checked{ color: red; }</string>
        </property>
        <property name="text">
         <string>Удалить входные файлы</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="RewriteButton">
          <property name="text">
           <string>Перезаписывать копии</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="SaveButton">
          <property name="text">
           <string>Сохранять копии</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetFixedSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Константа</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ConstantInput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>0x0</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Операция</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ORButton">
          <property name="text">
           <string>OR</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="XORButton">
          <property name="text">
           <string>XOR</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ANDButton">
          <property name="text">
           <string>AND</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="NOTButton">
          <property name="text">
           <string>NOT</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QRadioButton" name="ExactLaunchButton">
        <property name="text">
         <string>Разовый запуск</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="TimerButton">
        <property name="text">
         <string>По таймеру</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="TimerCount">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::Section::SecondSection</enum>
        </property>
        <property name="displayFormat">
         <string>H:mm:ss</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="LaunchButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Запуск</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="StopButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Стоп</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
